src/lib/db/query.js

```diff
@@ -44,10 +47,10 @@ const sqlBuilder = (obj) => {
 module.exports = {
   usageByDateType: (from, to, group) => db.manyOrNone(
     sqlBuilder({
-      'SELECT': 'date_time::DATE AS date, dataset, count(*)::INTEGER',
+      'SELECT': `${DTZ} AS date, dataset, count(*)::INTEGER`,
       'FROM': 'lev_audit',
       'WHERE': [from && fromDate, to && toDate, group && searchGroup],
-      'GROUP BY': 'date_time::date, dataset ORDER BY date_time::date'
+      'GROUP BY': `${DTZ}, dataset ORDER BY ${DTZ}`
```

Comment:
You don't need to use the `DTZ` here, and in fact it's probably not good practise... instead, you should be able to do the following:
```sql
GROUP BY 1, 2 ORDER BY 1
```

Reply:
OK done